core/unix/include/syscall_mach.h

```diff
@@ -38,6 +38,7 @@
 #define _SYSCALL_MACH_H_ 1
 
 #define SYSCALL_NUM_MARKER_MACH 0x1000000
+#define SYSCALL_NUM_MARKER_UNIX 0x2000000
```

Comment:
Would be good to update os_normalized_sysnum() (and elsewhere?) to use the named constant.

Reply:
Done.